tfhe/src/core_crypto/algorithms/lwe_multi_bit_programmable_bootstrapping.rs

```diff
@@ -289,11 +294,12 @@ pub fn prepare_multi_bit_ggsw_mem_optimized<
 ///     "Multiplication via PBS result is correct! Expected 6, got {pbs_multiplication_result}"
 /// );
 /// ```
-pub fn multi_bit_blind_rotate_assign<Scalar, InputCont, OutputCont, KeyCont>(
+pub fn multi_bit_ms_blind_rotate_assign<Scalar, InputCont, OutputCont, KeyCont>(
```

Comment:
This does not match what we have for the PBS, I don't like the fact we mod switch the lwe ciphertext entirely once, as we will be walking the lwe ciphertext once to do the transformation and again afterwards to do the PBS

Reply:
What do you don't like about it ?

Reply:
it becomes inconsistent with the existing PBS API and the we need to go through the whole LWE once, which represents ~ 800 * 8 = 6400 bytes of memory, while it could be done during the PBS as it's one today